UKHomeOffice/sgmr-service
# SPCR-387 Remove person from voyage

### AC / Description of changes
Adds remove person function to the people manifest on the voyage flow
Updated colours in stylesheet to use GOV UK names rather than hardcoded colours


### To Test
- create a voyage
- add people and get to the people manifest page
- click remove next to a person
- > screen should refresh and person should be removed
- go back to people selector page
- > person should be selectable again



### Notes

---

**Remember to merge to feature branches not `master`.**
